internal/config/config.go

```diff
@@ -0,0 +1,132 @@
+package config
```

Comment:
This file is mostly copy-paste from InfluxDB, there are some methods that aren't used (yet). The unused pieces will be used as part of implementing `config` and `setup`